ui/app/controllers/settings/tokens.js

```diff
@@ -52,9 +51,11 @@ export default class Tokens extends Controller {
   @action
   verifyToken() {
     const { secret } = this;
+    this.clearTokenProperties();
```

Comment:
`question`: This is action dispatches a few asynchronous actions via `ember-concurrency` and `ember-data`. But, I'm noticing that there is no `catch` for any failures or waiting until these actions have completed to set the secret on the token. How should we think about this? Is this a non-issue because some JavaScript magic?